pyproject.toml

```
@@ -46,13 +46,14 @@ dependencies = [
   "pandas",
   "pysat >= 3.1",
```

```suggestion
  "pysat >= 3.2",
```

Because of the header flags, no?